backend/src/db.ts

```diff
@@ -29,4 +30,47 @@ export class DBS{
             console.log('GET result ->' + res);
         });
     }
+
+    // Create Room
+    createRoom(roomid:any, roominfo:any){
```

Comment:
we should change roomid/roominfo to appropriate types in the future instead of any

Reply:
for roominfo, should the interface generally be in the same doc or split into another one as good practice for typescript?

Reply:
also, still not entirely sure what we want in the room information object

Reply:
> for roominfo, should the interface generally be in the same doc or split into another one as good practice for typescript?

you can do either for now once we have to more stuff we we'll have to probably reorganize again

> also, still not entirely sure what we want in the room information object
let's just make it an interface with room name and room description for now